packages/wheelhouse-core/src/development/developmentReducer.js

```diff
@@ -52,6 +85,31 @@ export default function(state = initialState, action) {
       expectedAction: "[17.015ms] About to convert to expected version" };
     return Object.assign({}, state, {logs: [...state.logs, newObject]});
   }
+  if (action.type === DEVELOPMENT_ENV_CHANGE) {
```

Comment:
Commenting in here is a bit excessive, let's take some of this out before merging it in